Makefile

```diff
@@ -6,14 +6,14 @@ setup:
 
 .PHONY: lint
 lint:
-	isort --check-only --diff neuro_extras tests setup.py
+	isort -rc --check-only --diff neuro_extras tests setup.py
```

Comment:
what do `-rc` params mean? (-c is `--check-only`, but I cannot find the explanation for `-r`)

Reply:
`-rc` stands for `recursive`